boards/ct/ctcc/ctcc_nrf52840.dts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2024 CTHINGS.CO
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf52840_qiaa.dtsi>
+
+/ {
+	model = "CTHINGS.CO Connectivity Card nRF52840";
+	compatible = "ct,ctcc-nrf52840";
+
+	chosen {
+		zephyr,console = &cdc_acm_uart;
+		zephyr,shell-uart = &cdc_acm_uart;
+		zephyr,sram = &sram0;
```

Comment:
(non blocking) `zephyr,uart-mcumgr = &cdc_acm_uart;`, `zephyr,bt-mon-uart = &cdc_acm_uart;` and `zephyr,bt-c2h-uart = &cdc_acm_uart;` could be added

Reply:
added, thanks